Add 2023-09-20 agenda and minutes (#571)
solid/specification: meetings/2023-09-20.md

* Join queue to talk.
* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.

### Participation and Code of Conduct
* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
* If this is your first time, welcome! please introduce yourself.

### Scribes
* Sarven Capadisli

### Introductions
* name: text

---

## Topics

### WIP Implementation Feedback
* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.

> * ML: a draft of Solid client-to-client standard for the Data Food Consortium project is published [here](https://datafoodconsortium.gitbook.io/dfc-standard-documentation/solid-client-protocol).
> * ML: How resources are expressed and where they should be stored. Also  few recommendations on indexing. Using DFC ontology and taxonomy. We're using type index to let the use choose where they  want to store the data.

Reviewer: ```suggestion
* ML: A draft of Solid client-to-client standard for the Data Food Consortium project is published [here](https://datafoodconsortium.gitbook.io/dfc-standard-documentation/solid-client-protocol).
  * ... It covers how resources are expressed and where they should be stored. Also, a few recommendations on indexing, using DFC ontology and taxonomy. We're using type index to let the user choose where they  want to store the data.
```

> * eP: What would you like to do re work items or work independently and provide feedback to the CG?
> * ML: Will let you know after.
> * eP: I'd be interested in it as a work item. I was working on Value Flows .. so interesting for me to see client-client for business domains, how much can be specific or generalised.

Reviewer: ```suggestion
* eP: What would you like to do — re-work items, or work independently and provide feedback to the CG?
* ML: Will let you know after.
* eP: I'd be interested in it as a work item. I was working on Value Flows ... so interesting for me to see client-client for business domains, how much can be specific or generalised.
```

* ML: Talked with Value Flows. Chose to use our own ontology for now. Will let you know. Open to discussion.

### Special Topic Meetings
URL: https://github.com/solid/specification/discussions/555

> * SC: A couple of items may need a date.
> * SC: "Security flaws in OIDC spec" still seems important. RH was/is? engaged w/ eP, HS. Why isn't this topic more up-front and centre in issues/chats?
> * SC: We have joint meeting scheduled with WebAgents CG on 2023-09-26. Please propose topics to the [agenda](https://hackmd.io/ftVbmvj0R0Oh6EsPWTEkwQ).
> * eP: There is a dedicated issue for OIDC we can link to it in 555. https://github.com/solid/solid-oidc/issues/221

Reviewer: ```suggestion
* SC: A couple of items may need a date.
  * ... "Security flaws in OIDC spec" still seems important. RH was/is? engaged w/ eP, HS. Why isn't this topic more up-front and centre in issues/chats?
  * ... We have a joint meeting scheduled with WebAgents CG on 2023-09-26. Please propose topics to the [agenda](https://hackmd.io/ftVbmvj0R0Oh6EsPWTEkwQ).
* eP: There is a [dedicated issue for OIDC](https://github.com/solid/solid-oidc/issues/221). We can link to it from [555](https://github.com/solid/specification/discussions/555).
```

### Solid Ecosystem Monitor
URL: https://github.com/virginiaBalseiro/solid-ecosystem-monitor/

* SC: View at https://virginiabalseiro.github.io/solid-ecosystem-monitor/

> * SC: Now includes data/charts for Present and Scribes (due to popular demand) from minutes.
> * SC: Please use same name in Present/Scribes as it appears in https://www.w3.org/groups/cg/solid/participants/ .

Reviewer: ```suggestion
  * ... Now includes data/charts for Present and Scribes (due to popular demand) from minutes.
  * ... Please use name in Present/Scribes as it appears in https://www.w3.org/groups/cg/solid/participants/ .
```

### Solid Protocol Version 0.11.0
URL: https://github.com/solid/specification/milestone/7

* SC: Let's make sure to add any missing issues/PRs that can reasonably make it into this release. The ED includes class 3 and higher changes, and some in the pipeline. See [Solid Protocol ED Changelog](https://solidproject.org/ED/protocol#changelog).

### Solid Symposium

> * RH: Solid event in March (follow up from Solid Symposium earlier this year.)
> * RH: What would for the community in that session? If we had tracks specific for developers working on the specs.
> * SC: Have people form the CG to guide/tutorial perhaps?
> * RH: Sure.
> * eP: 
> * RH: Details on Symposium. 2-3 May 2024 in Leuven. Mixed Industry, Academics & Developers. Chair: Bart Buelens at VITO. Previous edition is here: https://solid.ti.rw.fau.de/public/2023/solid-symposium/
> * RH: Organised in tracks, so it's possible to have a dedicated track for standards, making it a focal point for meeting in Europe in May 2024. Tracks may be hybrid. Another possibility, is to have a keynote speaker or two from the Solid CG community.

Reviewer: ```suggestion
* RH: Solid event in March (follow-up from Solid Symposium earlier this year.)
  * ... What would work for the community in that session? If we had tracks specific for developers working on the specs?
* SC: Have people from the CG to guide/tutorial perhaps?
* RH: Sure.
* eP: ???
* RH: Details on Symposium. 2-3 May 2024 in Leuven. Mixed Industry, Academics & Developers. Chair: Bart Buelens at VITO. Previous edition is here: https://solid.ti.rw.fau.de/public/2023/solid-symposium/
  * ... Organised in tracks, so it's possible to have a dedicated track for standards, making it a focal point for meeting in Europe in May 2024. Tracks may be hybrid. Another possibility is to have a keynote speaker or two from the Solid CG community.
```

### CG Chairs Election
URL: https://www.w3.org/community/solid/charter/#chairs

> * SC: Am looking into tooling. Trying to get support from W3C first. There are third-party tools/services but prefer to use W3C infra for simplicity and verification.
> * SC: Please contribute to [Self-Review Questionnaire for Chair Candidates](https://github.com/solid/specification/discussions/568). Mentioned at TPAC in [Chartering at W3C breakout session](https://github.com/w3c/tpac2023-breakouts/issues/43) ([minutes](https://www.w3.org/2023/09/13-w3process-minutes.html)), and it was well received towards WG chair selection considerations.

Reviewer: ```suggestion
* SC: I am looking into tooling. Trying to get support from W3C first. There are third-party tools/services, but prefer to use W3C infrastructure for simplicity and verification.
  * ... Please contribute to [Self-Review Questionnaire for Chair Candidates](https://github.com/solid/specification/discussions/568). Mentioned at TPAC in [Chartering at W3C breakout session](https://github.com/w3c/tpac2023-breakouts/issues/43) ([minutes](https://www.w3.org/2023/09/13-w3process-minutes.html)), and it was well received towards WG chair selection considerations.
```

### Replace panels with breakout task forces
URL: https://github.com/solid/process/issues/324

> * SC: https://github.com/solid/process/issues/324#issuecomment-1719720918
> * SC: Any objections to merge those PRs? Perhaps update chat URLs to use Matrix (replacing Gitter).
> * SC: Status update: https://matrix.to/#/!QxZtVBYQfMeMTnespj:gitter.im/$TetcgigSzuYAAIXt_I_hU8Po3ey6NAEPrgsl2QgO14Y
> * SC: webid-profile (TF) and test-suite-panel are no longer in the CG calendar. webid-profile convenes in chat before meeting. Note upcoming STM for Solid QA.
> * SC: data-interoperability-panel and notifications-panel have tentative status in the CG calendar.
> * RG: Do you have a timeline for when that transition will happen? Suppose you want to propose a new TF.. when can I do that?
> * SC: Any time but it is lightweight with specific focus.
> * eP: There is a STM.

Reviewer: ```suggestion
* SC: https://github.com/solid/process/issues/324#issuecomment-1719720918
  * ... Any objections to merging those PRs? Perhaps update chat URLs to use Matrix (replacing Gitter)?
  * ... Status update: https://matrix.to/#/!QxZtVBYQfMeMTnespj:gitter.im/$TetcgigSzuYAAIXt_I_hU8Po3ey6NAEPrgsl2QgO14Y
  * ... webid-profile (TF) and test-suite-panel are no longer in the CG calendar. webid-profile convenes in chat before meeting. Note upcoming STM for Solid QA.
  * ... data-interoperability-panel and notifications-panel have tentative status in the CG calendar.
* RG: Do you have a timeline for when that transition will happen? Suppose I want to propose a new TF... when can I do that?
* SC: Any time, but it is lightweight with specific focus.
* eP: There is a STM.
```
